spanner/spanner_snippets/spanner/spanner_create_client_with_query_options.go

```diff
@@ -30,7 +30,10 @@ import (
 func createClientWithQueryOptions(w io.Writer, database string) error {
 	ctx := context.Background()
 	queryOptions := spanner.QueryOptions{
-		Options: &sppb.ExecuteSqlRequest_QueryOptions{OptimizerVersion: "1"},
+		Options: &sppb.ExecuteSqlRequest_QueryOptions{
+			OptimizerVersion:           "1",
+			OptimizerStatisticsPackage: "auto_20191128_14_47_22UTC",
```

Comment:
Will this value always be accepted by the backend? I agree that it would be best to show an actual value here, instead of only showing how to set something like `latest`, but I don't know if this will fail during testing on (some) databases.

Reply:
I think it will always be valid. I replicated this from https://github.com/googleapis/java-spanner/pull/1239.

Reply:
Good point, it worked there, so it should work here as well.

Reply:
Hmmm.... It seems that this (sometimes) goes wrong: https://github.com/googleapis/java-spanner/issues/1273#issuecomment-879742347

Reply:
Interesting. It's supposed to always have it.